src/ops/dataflow.rs

```diff
@@ -183,14 +180,12 @@ pub struct CallIndirect {
 impl_op_name!(CallIndirect);
 
 impl DataflowOpTrait for CallIndirect {
+    const TAG: OpTag = OpTag::FnCall;
```

Comment:
Seems fishy that `Call` and `CallIndirect` both have the same tag, but I guess the latter operation is disappearing shortly anyway so I'm not too worried

Reply:
Yeah, tags are sets of operations so it's technically correct. But we've only added them as-needed.